### DISCUSSION TITLE
NPM init detect license file

### DISCUSSION THREAD

**Discussion by insprac** [npm/rfcs#307] [RRFC]
It would be nice if the `npm init` command detected the `LICENSE` file created by GitHub when creating a new repository and had it as the default.

**Comment by ljharb** [comment]
That is an incredibly good idea! (assuming the library github uses to derive a license identifier is publicly available)

**Reply by ruanmartinelli** [reply]
GitHub apparently uses [Licensee](https://github.com/licensee/licensee) for that ([source](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/licensing-a-repository#detecting-a-license)).

**Reply by ljharb** [reply]
ah, unfortunately the licensee npm already uses is a different project. a ruby gem isn't really a viable option for npm :-/
